Net24Solution/Net24Task/WebSiteData.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net.NetworkInformation;
+using System.Net.Mail;
+using System.Net;
+using System.Dynamic;
+using System.Timers;
+
+namespace Net24Task
+{
+    public class WebSiteData
```

Comment:
This class mixes configuration for WebSite and WebSite by itself.
Extract WebSiteConfig class  for storing settings